core/iwasm/compilation/aot_llvm_extra.cpp

```diff
@@ -41,6 +41,7 @@
 #include <llvm/Target/CodeGenCWrappers.h>
 #include <llvm/Target/TargetMachine.h>
 #include <llvm/Target/TargetOptions.h>
+#include <llvm/TargetParser/Triple.h>
```

Comment:
The `llvm/TargetParser/Triple.h` was introduced since llvm-17, and `llvm/ADT/Triple.h` was used before llvm-17, which is included in this file, see L11. How about adding
```C
#if LLVM_VERSION_MAJOR >= 17
#include <llvm/TargetParser/Triple.h>
#endif
```

Reply:
Done, thanks again